src/loader.js

```diff
@@ -253,6 +225,35 @@
     }
   }
 
+  // 4.4.3
+  Registry.prototype.lookup = function(key) {
+    return getRegistryEntry(this, key);
+  };
+
+  // 4.4.4
+  Registry.prototype.install = function(key, module) {
+    if (typeof this !== 'object')
+      throw new TypeError('registry must be an object');
+    if (this._registry.registryData[key])
+      throw new TypeError('Module with key ' + key + ' already exists');
+    if (!(module instanceof Module))
+      throw new TypeError('module must be an instance of Module');
```

Comment:
The checks on lines 235 and 237 are in the spec, but the check on line 239 is not. The reason I put it in here was because the same check was in the Loader.prototype.install function and it seemed to make sense. However, as I think about that reasoning, it's probably best just to follow the spec exactly so I can change this.

For my own reference later on, http://whatwg.github.io/loader/#registry-prototype-install